addons/van_dml/example/README.md

```diff
@@ -4,9 +4,14 @@ This is a  demo showing the design pattern and the usage of the commands introdu
 
 To use van_dml, include the script right after vanJS (an ES6-module will come soon)
```

Comment:
vanJS => VanJS

Reply:
Just out of curiosity, not trying to be rude......Why cant you (Tao-VanJS) make the changes you point out instead of waiting for and expecting the original author (efpage) to make the changes?  Is this a git thing or what?

Reply:
Hi @PAEz,

For your question, I can technically make the changes, but VanJS's 3rd party add-ons belong to their respective authors. While I can comment on things that I think would make sense to change, it would be better for the author to confirm the change is safe and what they want. Thus, I will only consider making changes myself after the author is unresponsive for a really long period of time.

Reply:
Ah thanks, makes perfect sense in this situation.  Was just wondering because I encounter this a lot elsewhere.

Reply:
I tried to resove any issues a.s.a.p.,but if after any reaction another typo is found, this is pretty time consuming. I do not think there is any risk of a safety issue from this; vanJS => VanJS, so I kindly ask you just to focus on relevant issues.